http/headers/feature-policy.json

```diff
@@ -194,7 +170,7 @@
                 "version_added": "79"
               },
               "firefox": {
-                "version_added": "65",
+                "version_added": "74",
```

Comment:
I'm not sure about removing the "dom.security.featurePolicy.header.enabled" flag from the parent but leaving it here with a bumped version. Can you back out these changes and leave them for https://github.com/mdn/browser-compat-data/issues/15987?